infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/DoubleTypeDefinition.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.json.types;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import java.io.IOException;
+import java.util.Objects;
+
+class DoubleTypeDefinition extends PrimitiveTypeDefinition<Double> {
+
+  @Override
+  public void serializeOpenApiTypeFields(final JsonGenerator gen) throws IOException {
+    gen.writeStringField("type", "number");
+  }
```

Comment:
check in the string integer type how we write the subtype (format) - we can set format to `double` looking at the spec..
https://swagger.io/specification/

Reply:
Added a format field.